integration/snapshot_test.go

```diff
@@ -42,17 +42,17 @@ func TestSnapshotGeneric(t *testing.T) {
 	// create new graph def
 	graphs := createCustomGraph()
 
-	graph_def, err := Jsonify(graphs[0].Definition)
+	graphDef, err := Jsonify(graphs[0].Definition)
```

Comment:
`graphDefinition` or `g` as its short lived? I prefer `graphDefinition` but Go devs seem to prefer short rather than long if the variable is short lived. See https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
I can go with g